packages/default/scss/dropdowntree/_layout.scss

```diff
@@ -5,19 +5,17 @@ $dropdowntree-popup-spacer-y: $padding-y-lg !default;
 
     // Dropdowntree popup
     .k-popup {
+
         .k-check-all {
             margin: $dropdowntree-popup-spacer-y $dropdowntree-popup-spacer-x 0;
         }
-        .k-check-all + .k-treeview {
+
+        .k-treeview {
```

Comment:
This way every treeview inside a popup will get the dropdowntree popup spacing, which is incorrect. The previous selector addressed that to target the dropdowntree specifically.

Reply:
There are scenarios without check all, so the previous selector did not meet our expectations. Now we added a more specific class name to the k-popup, so I used it for a better selector.

Reply:
The dropdowntree is a new widget, so it should not be with the "fix" prefix.

Reply:
Indeed, the DropDownTree is not a new widget. However, this PR fixes several other issues. Namely, these two issues should be marked as fixes, as they affect all components, not only the DropDownTree:

- fix right filter padding in RTL
- add RTL support for styles checkboxes and radiobuttons

It seems that the new selector is not available in the tests.